packages/react-reconciler/src/ReactFiberTreeReflection.js

```diff
@@ -346,8 +355,10 @@ function traverseFragmentInstanceChildren<A, B, C>(
       child.memoizedState !== null
     ) {
       // Skip hidden subtrees
+    } else if (skipPortals && child.tag === HostPortal) {
+      // Skip portals
```

Comment:
Optionally skip portal components during child traversal.
Idea being that for layout related instance methods, we don't want to special case their children as child host targets for the fragment. But for event registration we would.